libr/include/r_magic.h

```diff
@@ -264,6 +264,8 @@ struct r_magic_set {
 	/* FIXME: Make the string dynamically allocated so that e.g.
 	   strings matched in files can be longer than MAXstring */
 	union VALUETYPE ms_value;	/* either number or string */
+
+	ut32 last_cont_level;
```

Comment:
i think this variable should be initialized to 0 in the `file_apprentice()` function  (apprentice.c:280). im not 100% sure about the use of this variable, but as long as changing the RMagicSet struct implies changing the ABI, this PR may be hold until 5.7.9 (1 release away), so i think it's a good time to squash other variables in the same PR related to rmagic, this way we break abi only once

Reply:
Might try to incorporate changes from pr #20474 or just hold until 5.7.9